projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -82,16 +86,40 @@ export default function PublicizeForm( {
 							</Notice>
 						</PanelRow>
 					) }
+					{ brokenConnections.length > 0 && (
+						<Notice type={ 'error' }>
+							{ createInterpolateElement(
+								_n(
+									'One of your social connections is broken. Reconnect them on the <fixLink>connection management</fixLink> page.',
+									'Some of your social connections are broken. Reconnect them on the <fixLink>connection management</fixLink> page.',
+									brokenConnections.length,
+									'jetpack'
+								),
+								{
+									fixLink: <ExternalLink href={ connectionsAdminUrl } />,
+								}
+							) }
+						</Notice>
+					) }
 					<PanelRow>
 						<ul className={ styles[ 'connections-list' ] }>
 							{ connections.map(
-								( { display_name, enabled, id, service_name, toggleable, profile_picture } ) => (
+								( {
+									display_name,
+									enabled,
+									id,
+									service_name,
+									toggleable,
+									profile_picture,
+									is_healthy,
+								} ) => (
 									<PublicizeConnection
 										disabled={
 											( isRePublicizeFeatureEnabled ? ! isPublicizeEnabled : ! toggleable ) ||
-											( ! enabled && toggleable && outOfConnections )
+											( ! enabled && toggleable && outOfConnections ) ||
+											false === is_healthy
 										}
-										enabled={ enabled && ! isPublicizeDisabledBySitePlan }
+										enabled={ enabled && ! isPublicizeDisabledBySitePlan && false !== is_healthy }
```

Comment:
Nitpick, but this reads a little awkward. Isn't `false !== is_healthy` the same as just `is_healthy`?

Reply:
Reading back this might actually be intended, so feel free to ignore!

Reply:
Yeah we need to check it's actually `false`